feat: increase CSS specificity of components (#1766)
razorpay/blade: .changeset/tiny-pugs-sort.md

---
"@razorpay/blade": minor
---

feat: increase the CSS specificity of all components

> [!WARNING]
>

> > **We've enhanced the specificity of styles for our components. If you've defined styles in a separate CSS file targeting blade components, please note that they may no longer take effect as expected.

Author: ```suggestion
> **We've enhanced the specificity of styles for Blade components. If you've defined styles in separate CSS at a root level targeting blade components, please note that they may no longer take effect as expected.
```
